packages/office-ui-fabric-react/src/components/Dialog/Dialog.base.tsx

```diff
@@ -96,10 +97,26 @@ export class DialogBase extends React.Component<IDialogProps, {}> {
       mergedLayerProps.onLayerDidMount = onLayerDidMount;
     }
 
+    let dialogDraggableClassName: string | undefined;
+    let dragOptions: IDragOptions | undefined;
+
+    // if we are draggable, make sure we are using the correct
+    // draggable classname and selectors
+    if (modalProps && modalProps.dragOptions && !modalProps.dragOptions.dragHandleSelector) {
+      dialogDraggableClassName = 'ms-dialog-draggable-header';
+      dragOptions = {
+        ...modalProps.dragOptions,
+        dragHandleSelector: `.${dialogDraggableClassName}`
```

Comment:
Can't the `dialogDraggableClassName` string be in-lined? I don't see the benefit in having it be a local variable, perhaps I'm not following the diff well.

Reply:
I see.. it is the fact that you optionally pass it thru to `getMergedDialogContentStyles`... perhaps it is unavoidable then.. no worries.